sdk/servicebus/service-bus/src/sender.ts

```diff
@@ -109,6 +112,7 @@ export class Sender {
    * You will need the sequence number if you intend to cancel the scheduling of the message.
    * Save the `Long` type as-is in your application without converting to number. Since JavaScript
    * only supports 53 bit numbers, converting the `Long` to number will cause loss in precision.
+   * @throws {Error} Thrown if the underlying connection, client or sender is closed.
```

Comment:
Methods in the receiver.ts class that are implemented using the ManagementClient class have their docs updated to include the MessagingError. 

Any reason not to do the same for the below methods in the sender.ts class which fall under the same category?
- scheduleMessage
- scheduleMessages
- cancelScheduledMessage
- cancelScheduledMessages

Reply:
With https://github.com/Azure/azure-sdk-for-js/pull/6088/commits/2f628eda6e1c07f130760e2b4fb0725abb935d41, `MessagingError` is now documented for the above 4 cases, but I guess the same message has been copy/pasted from the one used in the send operation

> Thrown if any error occurs while sending messages to the service.

Please update the to reflect the right operation like `scheduleMessage`, `cancelScheduleMessage` etc.

Reply:
Doesn't seem to be an issue to me since these are operations on the sender, can update to include name of operation..